packages/bitcore-client/bin/wallet-create

```diff
@@ -10,11 +10,7 @@ program
   .version(require('../package.json').version)
   .option('--name <name>', 'REQUIRED - Wallet Name')
   .option('--chain <chain>', 'REQUIRED - Chain (BTC/BCH/ETH, etc)')
-  .option(
-    '--network <network>',
-    'REQUIRED - Network (livenet/testnet/regtest, etc)'
-  )
-  .option('--addressType [type]', 'optional - Specify the address type (default: pubkeyhash)')
```

Comment:
It seems that addressType was accidentally removed.

Reply:
good catch